pkg/cmd/admin/network/isolate_projects.go

```diff
@@ -9,6 +9,8 @@ import (
 	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
 	kerrors "k8s.io/kubernetes/pkg/util/errors"
 
+	osdnapi "github.com/openshift/openshift-sdn/plugins/osdn/api"
```

Comment:
I think we don't want pkg/cmd to depend on plugins/, please check https://github.com/openshift/openshift-sdn/pull/233